packages/scikit-image/index.rst

```diff
@@ -224,26 +216,26 @@ unsigned.
     are supposed to lie in [-1, 1] (in order to have comparable contrast for
     all float images) ::
 
-        >>> camera_float = util.img_as_float(camera)
+        >>> from skimage import img_as_float
+        >>> camera_float = img_as_float(camera)
         >>> camera.max(), camera_float.max()
-        (Image(255, dtype=uint8), 1.0)
+        (255, 1.0)
 
 Some image processing routines need to work with float arrays, and may
 hence output an array with a different type and the data range from the
 input array ::
 
-    >>> from skimage import filter
-    >>> camera_sobel = filter.sobel(camera)
-    >>> camera_sobel.max()
-    0.8365106670670005
+    >>> camera_sobel = filters.sobel(camera)
+    >>> camera_sobel.max() # doctest: +SKIP 
+    0.591502...
 
 
-Utility functions are provided in :mod:`skimage.util` to convert both the
+Utility functions are provided in :mod:`skimage` to convert both the
 dtype and the data range, following skimage's conventions:
 ``util.img_as_float``, ``util.img_as_ubyte``, etc.
 
 See the `user guide
-<http://scikit-image.org/docs/0.8.0/user_guide/data_types.html>`_ for
+<http://scikit-image.org/docs/dev/user_guide/data_types.html>`_ for
```

Comment:
I think that linking to a dev documentation is a bad idea, you could link to http://scikit-image.org/docs/dev/user_guide/data_types.html